condition/src/test/java/com/example/test/pages/KBBMOOCProject/CarForSaleConditionTabComponent.java

```diff
@@ -0,0 +1,84 @@
+package com.example.test.pages.KBBMOOCProject;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+
+public class CarForSaleConditionTabComponent extends KbbPageBase{
+
+    public CarForSaleConditionTabComponent(WebDriver driver) {
+        super(driver);
+        //TODO Auto-generated constructor stub
+    }
+    private WebElement conditionTab = keyword.findElement(By.xpath("//span[contains(text(),'Condition')]"));
+    private WebElement optionNew = keyword.findElement(By.xpath("(//div[normalize-space()='New'])[1]"));
+    private WebElement optionUsed = keyword.findElement(By.xpath("(//div[normalize-space()='Used'])[1]"));
+    private WebElement optionManufacturerCertified = keyword.findElement(By.xpath("//div[normalize-space()='Manufacturer Certified']"));
+    private WebElement optionThirdPartyCertified = keyword.findElement(By.xpath("//div[contains(text(),'Third-Party Certified')]"));
+    private WebElement titleModuleYourSearch = keyword.findElement(By.xpath("//span[@class='text-size-400 text-bold text-gray-base']"));
+
+    /**
+     * Method verify that Year Tab is displayed Correct
+     * @return true or false
+     */
+    public boolean isTheConditionFilterShouldBeDisplayedCorrectWhenUserClickOnIt(){
+        keyword.scrollToElement(conditionTab);
+        if (isElementDisplayed(optionNew) == true && isElementDisplayed(optionManufacturerCertified) == true && 
+        isElementDisplayed(optionThirdPartyCertified) == true && isElementDisplayed(optionUsed))
+        {
+            return true;
+        }
+        else return false;
+    }
+
+    /**
+     * Check Three element is Displayed
+     * 
+     * @return
+     */
+    public boolean checkOptionIsDisplayedCorrect() {
+        keyword.scrollToElement(titleModuleYourSearch);
+        WebElement optionConditionYourSearch = keyword.findElement(By.xpath(

```

Comment:
should write the web element locator in separate line then call into a method/function